### ISSUE TITLE
In the fire menu, switch from aim and fire to just aim

### ISSUE THREAD

**Issue by BjoHart** [CleverRaven/Cataclysm-DDA#71311]
### Is your feature request related to a problem? Please describe.

To play with the bow, i can’t tell you how many times I’ve pressed the aim button and wished for it.

### Solution you would like.

How would it be if you could not shoot in the fire menu with the three selection options, but only aim. Then you should execute the shot manually.

Or if it were possible, let us switch between the two options (aim and fire) or just (aim).

![fire mode](https://github.com/CleverRaven/Cataclysm-DDA/assets/69985946/3806f0a7-4bdb-43ad-b3ee-1c7debd9e1b1)

### Describe alternatives you have considered.

None

**Comment by mqrause** [comment]
`[.] to steady your aim.  (10 moves)` Is aiming without shooting. You can spend any time you want with that and then shoot manually.

**Comment by BjoHart** [comment]
> `[.] to steady your aim. (10 moves)` Is aiming without shooting. You can spend any time you want with that and then shoot manually.

I know, I know.
I would like to have an option to get to the different target modes with one button press, without having to press [.] 15 times and then shoot...

**Comment by XygenSS** [comment]
pressing . already gives you plenty of granular control imo. You can hold the key down to spam it, and if you're waiting for a target to step closer to you the . button is the best

**Comment by BjoHart** [comment]
That's your opinion. Perhaps others have a different opinion.
It has often happened to me that when I hold down the button, the opponent comes too close and sometimes even hits me.
And constantly spamming a button is precisely the point that annoys me.
I would think it would be cool to be able to switch.

**Comment by mqrause** [comment]
> As the opponent gets closer, the target values also change. I want to be able to react precisely to this.
> That's why the idea with the three different options is not to shoot, but only to aim.
> If you press precise aim for an example and the target suddenly gets closer, the opponent might hit you, because you pressed the button for precise aim and couldn't react...

I'm not sure how your idea would help with targets getting too close? As I understand it, you want to have single button presses to reach regular/careful/precise aim, but doing that would have all the same issues, since it still passes larger amounts of time at once, just minus the time to actually fire.

**Comment by PatrikLundell** [comment]
I think I understand what @BjoHart is after (and what I would want as well):
- The target is "far" away, so I want to save on key press strain by getting rid of the first 20 or so "." key presses by spending time to aim precisely and then pressing "." to wait until the optimal moment.
- Guessing incorrectly and get hit while aiming is very annoying (and, of course, potentially deadly). It would be useful to get the option to interrupt the aiming process when an enemy gets dangerously close, but before it hitting you and wasting all the accumulated aiming. Obviously it should result in an aborted aiming if you haven't reached the basic aiming level (slow aiming weapons such as bows are particularly prone to this, although you currently break off only when hit). This functionality would allow you to fire off a shot with whatever aim you've managed to accumulate, or decide to immediately try to run or switch to a melee weapon.
Basically, what I'd want here is to be dropped back at the aiming menu for a decision to do single step aiming, firing, aborting, or gamble on you being lucky and try a longer aiming time option anyway. If you hadn't managed to get a basic aim you should be give the choice to continue until you get it or to abort the aiming.

There are cases where you want to aim and shoot as a single action, although they're rare in my case (I expect there are others who have a different play style, though).

**Comment by ashGlaw** [comment]
I agree. I've had the same thought in the past. I don't want to have to press 5 a bajillion times. This change would be a big quality of life improvement for me regarding shooting. Especially with a toggle between just aim and aim and shoot

**Comment by BjoHart** [comment]
> I think I understand what @BjoHart is after (and what I would want as well):
> 
> There are cases where you want to aim and shoot as a single action, although they're rare in my case (I expect there are others who have a different play style, though).

Thats what i mean. Sry for my englisch, im german ;-)

**Comment by kevingranade** [comment]
This issue has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/in-the-fire-menu-switch-from-aim-and-fire-to-just-aim/29128/2

**Comment by pjf** [comment]
I would love this so much, and @PatrikLundell's summary is spot-on.

This feature would also avoid the situation where the target moves behind blocking terrain or another creature, resulting in the player shooting something they didn't even intend to.

**Comment by BjoHart** [comment]
If someone can tell me where the file is where the changes need to be made, i give it a try...
@kevingranade

**Comment by BjoHart** [comment]
Or maybe it is easier if only two new buttons are introduced.
One for aiming 50 moves and one for aiming 100 moves.
This should definitely reduce the number of keystrokes for each battle.
